SKILL.tsv

```diff
@@ -921,114 +921,114 @@ SKILL_20150317_000920	Thrust
 SKILL_20150317_000986	Impaler
-SKILL_20150317_000987	{#DD5500}{ol}[Stab]{/}{/}{nl} I go to plug through the small monsters in the window. If you use the skill to take down to the ground after you plug in the window will increase.
+SKILL_20150317_000987	{#DD5500}{ol}[Stab]{/}{/}{nl} Impales a small monster with a spear. Can use this skill with a skewered monster to inflict damage.
 SKILL_20150317_000988	Earth Wave
-SKILL_20150317_000989	{#DDD300}{ol}[Strike]{/}{/}{nl}Gather strength and smash the ground to attack enemy.
-SKILL_20150317_000990	Trout
-SKILL_20150317_000991	Movement speed will increase.
+SKILL_20150317_000989	{#DDD300}{ol}[Strike]{/}{/}{nl} Gather up power and use your weapon to attack the enemy by smashing it to the ground.
+SKILL_20150317_000990	Trot
+SKILL_20150317_000991	Increases your movement speed.
 SKILL_20150317_000992	Movement Speed +#{CaptionRatio}#%{nl}Duration #{CaptionTime}# seconds
-SKILL_20150317_000993	Stead Charge
-SKILL_20150317_000994	{#DDD300}{ol}[Strike]{/}{/}{nl}Rush forward and push away enemy.
+SKILL_20150317_000993	Steed Charge
+SKILL_20150317_000994	{#DDD300}{ol}[Strike]{/}{/}{nl}Rush forward with power to throw your target away.
 SKILL_20150317_000995	Doom Spike
-SKILL_20150317_000996	{#DDD300}{ol}[Strike]{/}{/}{nl}Gather force and and strongly stab enemy.
+SKILL_20150317_000996	{#DDD300}{ol}[Stab]{/}{/}{nl} Gather up power to release a strong stab to an enemy.
 SKILL_20150317_000997	Rush
-SKILL_20150317_000998	{#DD5500}{ol}[Slash]{/}{/}{nl}Attack enemy in fron continuously while spinning the spear over head.
+SKILL_20150317_000998	{#DD5500}{ol}[Slash]{/}{/}{nl} Continuously spin your spear above your head to attack an enemy.
 SKILL_20150317_000999	Attack #{SkillAtkAdd}#{nl}Max. Duration 10 seconds{nl}Splash #{SkillSR}#
 SKILL_20150317_001000	Dragging Death
 SKILL_20150317_001001	{#DDD300}{ol}[Stab]{/}{/}{nl}Attacks enemies in front continuously.
 SKILL_20150317_001002	Embowel
-SKILL_20150317_001003	{#DD5500}{ol}[Stab]{/}{/}{nl} Kick the enemies after strongly stinging.
-SKILL_20150317_001004	Flange
-SKILL_20150317_001005	Continuously attack enemy with basic attack after using skill to increase ATK.
-SKILL_20150317_001006	Attack + #{CaptionRatio}#{nl}Max. overlap #{CaptionRatio2}# times {nl}Duration #{CaptionTime}# seconds
+SKILL_20150317_001003	{#DD5500}{ol}[Stab]{/}{/} {nl} Release a strong stab to an enemy while kicking it away afterwards.
+SKILL_20150317_001004	Frenzy
+SKILL_20150317_001005	Increases your attack when you use consecutive basic attacks on the target.
+SKILL_20150317_001006	Attack +#{CaptionRatio}# {nl} Stacks up to #{CaptionRatio2}# times {nl} Duration #{CaptionTime}# seconds
 SKILL_20150317_001007	Aggressor
-SKILL_20150317_001008	Attack is accurate but chance of critical and critical evasion decreases.
+SKILL_20150317_001008	Your attack is guaranteed to always hit, while your Critical Rate and Critical Evasion decreases.
```

Comment:
No need for "guaranteed" and "always" in the same line, can be easily shortened to "Your attacks are guaranteed to hit, but your Critical Rate and Critical Evasion decreases".